docs/developer/advanced/upgrading-nodejs.asciidoc

```diff
@@ -16,8 +22,8 @@ These files must be updated when upgrading Node.js:
   - {kib-repo}blob/{branch}/package.json[`package.json`] - The version is specified in the `engines.node` field (if possible, also upgrade `@types/node` to match the new version, both under `devDependencies` and `resolutions`).
   - {kib-repo}blob/{branch}/WORKSPACE.bazel[`WORKSPACE.bazel`] - The version is specified in the `node_version` property.
     Besides this property, the list of files under `node_repositories` must be updated along with their respective SHA256 hashes.
-    These can be found on the https://nodejs.org[nodejs.org] website.
-    Example for Node.js v18.17.1: https://nodejs.org/dist/v18.17.1/SHASUMS256.txt.asc
+    These can be found in the `SHASUMS256.txt` file inside the public `kibana-custom-node-artifacts` GCP bucket.
+    Example for Node.js v18.18.2: https://console.cloud.google.com/storage/browser/_details/kibana-custom-node-artifacts/node-glibc-217/dist/v18.18.2/SHASUMS256.txt[kibana-custom-node-artifacts/node-glibc-217/dist/v18.18.2/SHASUMS256.txt]
```

Comment:
Should private links be in the public facing docs?

Reply:
It's a public bucket / link. Anyone with a Google account can sign in a see it.

Reply:
The link has now been updated to point directly to the file so you don't have to be logged in to view it.